bitnami/kubeapps/templates/tests/test-chartsvc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-chartsvc-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-chartsvc-test
+    image: {{ .Values.testImage.repository }}:{{ .Values.testImage.tag }}
+    env:
+    - name: CHARTSVC_HOST
+      value: {{ template "kubeapps.chartsvc.fullname" . }}.{{ .Release.Namespace }}
+    - name: CHARTSVC_PORT
+      value: "{{ .Values.chartsvc.service.port }}"
+    command:
+    - sh
+    - -c
+    - curl $CHARTSVC_HOST:$CHARTSVC_PORT/v1/charts | grep wordpress
```

Comment:
This happens only if the import job has finished no? Is there any possibility for a race condition?

Reply:
Before creating any issue I'd like to know @andresmgot take on some of my comments since some of my concerns might not be applicable.

Reply:
yes, in the CI we wait for the import job to have finished in order to execute the helm tests (see my previous comment)